src/Validator/UnsignedRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Zumba\VanillaJsConnect\Validator;
+
+use Zumba\VanillaJsConnect\Response as Response;
+use Zumba\VanillaJsConnect as Vanilla;
+
+class UnsignedRequest implements \Zumba\VanillaJsConnect\ValidatorInterface
+{
+
+    public function validate(Vanilla\Request $request, Vanilla\User $user = null, Vanilla\Config $config = null)
+    {
+        $requestTimestamp = $request->getTimestamp();
+        $requestSignature = $request->getSignature();
+
+        if (!isset($requestTimestamp) && !isset($requestSignature)) {
```

Comment:
@j-feng13 I think these should be `empty` checks, rather than `!isset`. Some invalid falsey things like empty strings would fail this check: http://codepad.viper-7.com/7Rb6d8